sdk/src/main/java/com/hedera/hashgraph/sdk/ContractCreateFlow.java

```diff
@@ -0,0 +1,342 @@
+package com.hedera.hashgraph.sdk;
```

Comment:
Can we put this in a `flow` package to encapsulate and aid in discoverability? The sdk package is already big enough.

Reply:
I do agree that this package is big enough, but the reason we put everything in one package to begin with was because of Java's package-private scope issues and that's not an issue I want to resurface.

I just pulled this branch and tried to see if a `flow` package would work in the first place since I had doubts, and it doesn't since we need access to `ContractFunctionParams.toBytes()` which is package-private.